test/end2end/app_test.dart

```diff
@@ -64,6 +78,15 @@ void main() {
     await tester.pumpAndSettle();
   }
 
+  void goToPoint(GeoPoint point) {
```

Comment:
I just wonder if we could add a little line of documentation here.

Reply:
feb1b7e529e2a4dd9ef704dd89ea84a3ad25657c